### PR TITLE
changes

### PR DESCRIPTION
**Pull request by Favna**
commits = what is changed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ EXAMPLE
   hello world from ./src/hello.ts!
 ```
 
-_See code: [src/commands/hello.ts](https://github.com/cfanoulis/klara/blob/v0.0.0/src/commands/hello.ts)_
+_See code: [src\commands\hello.ts](https://github.com/cfanoulis/klasa-cli/blob/v0.0.0/src\commands\hello.ts)_
```
Comment (Favna) [inline README.md:53]:
```suggestion
_See code: [src/commands/hello.ts](https://github.com/cfanoulis/klara/blob/v0.0.0/src/commands/hello.ts)_
```

File: README.md [hunk 2]
```diff
@@ -67,5 +67,5 @@ OPTIONS
   --all  see all commands in CLI
 ```
 
-_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v3.0.1/src/commands/help.ts)_
+_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v3.0.1/src\commands\help.ts)_
```
Comment (Favna) [inline README.md:70]:
```suggestion
_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v3.0.1/src/commands/help.ts)_
```

File: package.json [hunk 3]
```diff
@@ -35,10 +35,9 @@
   "files": [
     "/bin",
     "/lib",
-    "/npm-shrinkwrap.json",
```
Comment (cfanoulis) [inline package.json:38]: please don't touch this I think oclif uses it for dependency management (idk why)

File: README.md [hunk 4]
```diff
@@ -16,11 +16,11 @@ Manage your klasa bot with ease!
 # Usage
 <!-- usage -->
 ```sh-session
-$ npm install -g klara
+$ npm install -g klasa-cli
 $ kl COMMAND
 running command...
 $ kl (-v|--version|version)
-klara/0.0.0 linux-x64 node-v14.3.0
+klasa-cli/0.0.0 win32-x64 node-v14.2.0
```
Comment (Favna) [inline README.md:23]:
```suggestion
klasa-cli/0.0.0 linux-x64 node-v14.2.0
```

File: package.json [hunk 5]
```diff
@@ -59,11 +58,11 @@
   },
   "repository": "cfanoulis/klasa-cli",
   "scripts": {
+    "lint": "eslint src --ext ts --fix",
+    "build": "tsc",
     "postpack": "rm -f oclif.manifest.json",
-    "posttest": "eslint . --ext .ts --config .eslintrc",
-    "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
```
Comment (cfanoulis) [inline package.json:64]: no, we need those 2

File: README.md [hunk 6]
```diff
@@ -16,11 +16,11 @@ Manage your klasa bot with ease!
 # Usage
 <!-- usage -->
 ```sh-session
-$ npm install -g klara
+$ npm install -g klasa-cli
```
Comment (Favna) [inline README.md:19]:
```suggestion
$ npm install -g klara
```

File: README.md [hunk 7]
```diff
@@ -16,11 +16,11 @@ Manage your klasa bot with ease!
...
 $ kl COMMAND
 running command...
 $ kl (-v|--version|version)
-klara/0.0.0 linux-x64 node-v14.3.0
+klasa-cli/0.0.0 linux-x64 node-v14.2.0
```
Comment (Favna) [inline README.md:23]:
```suggestion
klara/0.0.0 linux-x64 node-v14.2.0
```

**Pull request merged**